tests/cmds/conftest.py

```diff
@@ -93,3 +108,16 @@ def f(*args):
     f.call_count = 0
     f.call_args_list = []
     return f
+
+
+def get_generator_for_get_all(mocker, mock_return_items):
+    mock_return_items = mock_return_items or ""
+
+    def gen(*args, **kwargs):
+        response = mocker.MagicMock(spec=Request)
+        response.text = """{{"items": [{0}]}}""".format(mock_return_items)
+        pages = [response]
+        for page in pages:
+            yield Py42Response(page)
```

Comment:
You don't have to wrap a single item in a list here, you can just:

```
yield Py42Response(response)
```
and the generator will only have one item.